# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/nautilus/root-ok.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- nautilus-49.2/src/nautilus-main.c.vanilla	2025-12-04 19:31:33.460110093 +0200
+++ nautilus-49.2/src/nautilus-main.c	2025-12-04 19:31:42.237185226 +0200
@@ -61,7 +61,7 @@
         g_warning (_("Running as root is not supported. "
                      "Consider running `nautilus admin:///` instead."));
 
-        exit (ENOTSUP);
+        //exit (ENOTSUP);
     }
 
     nautilus_register_resource ();
